### DISCUSSION TITLE
Automation not detected Nuendo/Cubase

### DISCUSSION THREAD

**Discussion by djniviro** [apohl79/audiogridder#1104] [Help]
Hi all!

Ever since the beginning of using AudioGridder, I've never been able using any kinds of automation in Nuendo. This doesn't work in UI or generic mode and I have no idea what I do wrong. What I always do is selecting "write", playing it back and then moving the parameters, which result in the write of automation data. However nothing passes through.

Thanks for the help!

**Comment by apohl79** [comment]
You have toi assign the params you want to use: https://audiogridder.com/plugin/#automation
